improver_tests/utilities/test_pad_spatial.py

```diff
@@ -154,7 +154,7 @@ def setUp(self):
 
     def test_basic(self):
         """Test function returns a cube with expected metadata"""
-        halo_size_km = 162.0
+        halo_size_km = 2.0
```

Comment:
Both these tests could have a `self.halo_size_m = 2000.0`, to reduce duplication and get rid of the multiplying by 1000 in the calls.

Reply:
Updated in commit d779a5b3fadb1f062f0dcb1a0f4de01ad9346dc1.

Reply:
Modified in commit 732bd75c56959880c79c2dcee0f7b88b7c9ff1e8.

Reply:
Also moved grid_spacing and domain_corner setup to a separate function in commit 732bd75c56959880c79c2dcee0f7b88b7c9ff1e8. Could be useful if it ever needs to be changed to have it in one place.